integration/spark/src/main/java/org/apache/carbondata/spark/load/CarbonLoadModel.java

```diff
@@ -117,9 +117,9 @@
   private String badRecordsLoggerEnable;
 
   /**
-   * defines the option to specify the bad record log redirect to raw csv
+   * defines the option to specify the bad record logger action
    */
-  private String badRecordsLoggerRedirect;
+  private String badRecordsLoggerAction;
```

Comment:
This action is not for Logger, right? Perhaps `badRecordsAction` is a better name?
And it should be an enum instead of String